packages/mdc-toolbar/_mixins.scss

```diff
@@ -14,14 +14,38 @@
 // limitations under the License.
 //
 
+@import "@material/theme/variables"; // for mdc-theme-accessible-ink-color
 @import "./variables";
 
-/**
- * Applies styles to the different types of icons that can exist in toolbars.
- * Both .mdc-toolbar__icon and .mdc-toolbar__menu-icon share all styles except for
- * horizontal padding.
- */
-@mixin mdc-shared-icon-style() {
+//
+// Public
+//
+
+@mixin mdc-toolbar-ink-color($color) {
+  @include mdc-theme-prop(color, $color);
+}
+
+@mixin mdc-toolbar-fill-color($color) {
+  @include mdc-theme-prop(background-color, $color);
+}
+
+@mixin mdc-toolbar-fill-color-accessible($color) {
```

Comment:
Confused what it means to be accessible regarding color. Is this like high contrast mode? If so, that seems to me like another theme (like dark-theme).

Reply:
Kind of, but not quite. It's not a separate "mode", per se, but simply a convenience mixin for clients who want their button's fill (bg) color to be X, but don't care what the ink (text) color is as long as it's high enough contrast to pass accessibility requirements
([4.5:1 contrast ratio](https://webaim.org/resources/contrastchecker/) or higher).

Assuming that the client has accessible values for `primary` in `$mdc-theme-text-colors`, this mixin guarantees that the ink color it chooses will be accessible on the given fill color.

Reply:
That makes sense...should I also be doing this for drawer?

Reply:
Yes - generally speaking, all `-fill-color` mixins should also have a `-fill-color-accessible` version that sets the ink color automatically.